test/parallel/test-fs-watch-error.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
Please move this test to https://github.com/nodejs/node/blob/master/test/sequential/test-fs-watch.js